src/components/date-picker/flatpickr-adapter/flatpickr-adapter.scss

```
@@ -15,7 +15,11 @@
 
     * {
         box-sizing: border-box;
-        font-family: 'Roboto';
+        // To ensure that the date-picker's popover is font-agnostic,
+        // but still displays the same font as the rest of the components.
+        // This is needed, since the popover opens in the Portal,
+        // which inherits its font from the root element (Kompendium).
+        font-family: var(--kompendium-example-font-family, inherit);
```

this one still needs to be change.
```suggestion
        font-family: inherit;
```

---

Ah! I must have missed this one… 🤔

---

Ah, crap… I made a mistake when I removed my fixup. Never mind, I'll fix it 👍

---

⚡

---

We don't need the `font-family: inherit;`, as far as I can see. It works correctly without it in the docs at least, and it's what I'd expect, since `inherit` is the default behaviour anyway, right?